docker-qgis/qfc_worker/commands_base.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+from pathlib import Path
+from typing import Any
+
+from qfc_worker.workflow import Workflow, run_workflow
+
+logger = logging.getLogger(__name__)
+
+
+parser = argparse.ArgumentParser(prog="COMMAND")
+subparsers = parser.add_subparsers(dest="cmd")
```

Comment:
In theory these two can also go to the metaclass below, but failed to do it at first go. Since it is an implementation detal, let them live as global vars.